.github/workflows/ci-additional.yml

```diff
@@ -18,7 +18,8 @@ jobs:
       - name: Setup Conda Environment
         uses: conda-incubator/setup-miniconda@v2
         with:
-          miniconda-version: "latest"
+          mamba-version: "*"
```

Comment:
Probably useful to have a look at the master of the `setup-miniconda` actions and use `miniforge-variant: Mambaforge`. This gives you a base environment with mamba pre-installed and `conda-forge` as a channel.

Reply:
Doesn't work:
```
Warning: Unexpected input(s) 'miniforge-variant', 'miniforge-version', 'use-mamba', valid inputs are ['installer-url', 'miniconda-version', 'conda-version', 'conda-build-version', 'environment-file', 'activate-environment', 'python-version', 'add-anaconda-token', 'add-pip-as-python-dependency', 'allow-softlinks', 'auto-activate-base', 'auto-update-conda', 'condarc-file', 'channel-alias', 'channel-priority', 'channels', 'show-channel-urls', 'use-only-tar-bz2', 'remove-profiles', 'mamba-version', 'architecture']
```

Reply:
They are only yet available on `master`, not on `v2`. If they also work here well, `setup-miniconda` could do a new release to have them available in a stable fashion.

Reply:
(ignore; I was still using @v2)

Reply:
it works now, thanks